content/cybersecurity/concepts/disaster-recovery/disaster-recovery.md

```diff
@@ -0,0 +1,50 @@
+---
+
+**Disaster Recovery** is planning, implementing and following processes to restore IT systems and infrastructures during and after disruptive events.
+
+## The Importance Disaster Recovery
+
+Certain events can compromise or completely incapacitate IT systems. These disruptive events cannot always be prevented or controlled. They can be categorized as:
+
+- Natural Disasters (e.g., earthquakes, floods, hurricanes)
+- Human-Caused Disasters (e.g., terrorism, accidents, [cyber attacks](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack))
+- Other Incidents (e.g., power outages, equipment failure, infectious disease outbreaks)
+
+Disruptive events can have varying levels of severity and cause data loss, operational downtime, financial losses, and reputational damage to organizations and businesses. Organizations use Disaster Recovery procedures to reduce the impact of these adverse outcomes.
+
+## Disaster Recovery Plan
+
+The Disaster Recovery Plan is a document that organizations create and maintain to minimize the damage of disruptive events. It contains a complete analysis of the risks and the necessary actions to recover and restore IT systems in case of disruptive events. The Disaster Recovery Plan ensures that organizations can resume normal operations.
+
+The Disaster Recovery Plan contains the following elements:
+
+- Risk Analysis: It identifies potential hazards and risks of disruptive events.
+- Business Impact Analysis: It determines the identified risks' impact on the business and the critical data and infrastructures for the organization.
+- Recovery Point Objective: It specifies the backup timeframe that allows the organization to return to normal operation. It defines the frequency of the backups.
+- Recovery Time Objective: It defines the maximum downtime after a disruptive event that is not causing significant damage to the business.
+- Procedure: It contains actions that prevent the loss of critical data and a step-by-step guide on responding to different disruptive events and returning to normal operation.
+
+## The Types of Disaster Recovery
+
+Depending on the industry, criticality, IT infrastructure, and size of the organizations, different types of Disaster Recovery methods are suitable for them. The types of Disaster Recovery are the following:
+
+- Backup: Storing only critical data on a removable drive or off-site without backing up the IT infrastructure.
+- Network Disaster Recovery: Restoring network connectivity for internal and external communication, data sharing, and application access during a disaster.
+- Cloud Disaster Recovery: Using cloud services to host (automated) Disaster Recovery processes,  for example, automatic failover of workloads to another cloud platform in case of disruption.
+- Disaster Recovery as a Service (DRaaS): Outsourcing computer processing to a provider's cloud infrastructure to ensure business operations during disasters.
```

Comment:
[nitpick] I think it would be helpful to add links to products that offer cloud services. 
```suggestion
- Cloud Disaster Recovery: Using cloud services to host (automated) Disaster Recovery processes,  for example, automatic failover of workloads to another cloud platform in case of disruption.
- Disaster Recovery as a Service (DRaaS): Outsourcing computer processing to a provider's cloud infrastructure to ensure business operations during disasters.
```

Reply:
I added some links to the DRaaS section.